src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -399,13 +399,7 @@ public override bool IsReferenceType
         /// parameter, IsReferenceType and IsValueType will both return false.
         /// </summary>
         public override bool IsValueType
-        {
-            get
-            {
-                var kind = TypeKind;
-                return kind == TypeKind.Struct || kind == TypeKind.Enum;
-            }
-        }
+            => TypeKind is TypeKind.Struct or TypeKind.Enum or TypeKind.Extension;
```

Comment:
>=> TypeKind is TypeKind.Struct or TypeKind.Enum or TypeKind.Extension; [](http://example.com/codeflow?start=12&length=70)

Why is the style change?

Reply:
I am referring to the body

Reply:
No local declaration needed, so we no longer need a full body

Reply:
> No local declaration needed, so we no longer need a full body

In my opinion this constitutes an unnecessary style change. There is no requirement to switch to expression body whenever possible. This in not a question of whether something possible or not. This is a question personal preference. For example, I would prefer the original body style.